packages/generator-clay-component/app/templates/_package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "<%= repoName %>",
+	"version": "3.0.0",
+	"description": "<%= componentName %> component",
+	"license": "BSD-3-Clause",
+	"repository": "https://github.com/liferay/clay/tree/master/packages/<%= repoName %>",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "lib/<%= componentName %>.js",
+	"esnext:main": "src/<%= componentName %>.js",
+	"jsnext:main": "src/<%= componentName %>.js",
+	"files": ["lib", "src"],
+	"scripts": {
+		"compile": "tsc",
+		"prepublish": "npm run compile",
+		"start": "webpack-dev-server --mode development",
+		"test": "jest"
```

Comment:
We should probably build a ".d.ts" file as well (something like I am doing [here](https://github.com/wincent/js/blob/c65bc118839edfae0490313ad0090fe642a9d170/Makefile#L9) in my test side-project).